.github/workflows/ci.yml

```diff
@@ -498,8 +497,6 @@ jobs:
                 github.event.client_payload.pull_request.head.sha == github.event.client_payload.slash_command.args.named.sha &&
                 format('refs/pull/{0}/head', github.event.client_payload.pull_request.number) || '' }}
       - uses: ./.github/actions/setup
-        with:
-          cache: false
```

Comment:
Thanks for clarifying it's intentional.
Should it be a seprate commit with its own description so that I could understand why the change is being made?

Reply:
Sure, I can split it off.